app/validators/initiative_has_manager_validator.rb

```diff
@@ -0,0 +1,7 @@
+class InitiativeHasManagerValidator < ActiveModel::Validator
+  def validate(initiative)
+    unless initiative.source.is_a? Manager or initiative.helpers.any? ->(h){ h.is_a? Manager }
```

Comment:
Don't pass a lambda to `any?` it's more readable to use a proc i.e `initiative.helpers.any?  { |h| h.is_a? Manager }`

Reply:
P.D: A lambda is different from a proc, it is left to the reader to check on why

Reply:
Hum, why is a proc more readable?